package exceptionhandling;

public class Test2 {
	public static void main(String[] args) {
		/*int nums[] = new int[5];
		System.out.println(nums[9]);
//throws arrayindexoutofbound exception
//java created exception object based on the reason
		
*/
		
		
		
		
	}
	

}
